src/search/components/dateTime.component.tsx

```diff
@@ -14,30 +14,30 @@ export const datesEqual = (date1: Date | null, date2: Date | null): boolean => {
   return date1 !== null && date2 !== null && isEqual(date1, date2);
 };
 
-export interface UpdateFilterParams {
+export interface VerifyAndUpdateDateParams {
   date: Date | null;
   prevDate: Date | null;
   otherDate: Date | null;
   fromDateOrToDateChanged: 'fromDate' | 'toDate';
   changeDate: (date: Date | null) => void;
 }
 
-export function updateFilter({
+export function verifyAndUpdateDate({
   date,
   prevDate,
   otherDate,
   fromDateOrToDateChanged,
   changeDate,
-}: UpdateFilterParams): void {
+}: VerifyAndUpdateDateParams): void {
   if (
     date &&
     isValid(date) &&
-    !datesEqual(date, otherDate) &&
+    // !datesEqual(date, otherDate) &&
```

Comment:
Remove this commented line if it's not useful anymore

Reply:
Addressed in https://github.com/ral-facilities/operationsgateway/pull/138/commits/429360b8c1be31137242057ba69c5e9b465695a3